packages/twenty-server/src/core/quick-actions/intelligence.service.ts

```diff
@@ -0,0 +1,47 @@
+import { Injectable } from '@nestjs/common';
+
+import axios from 'axios';
+
+import { CompanyInteface } from 'src/core/quick-actions/interfaces/company.interface';
+
+import { EnvironmentService } from 'src/integrations/environment/environment.service';
+
+@Injectable()
+export class IntelligenceService {
+  constructor(private readonly environmentService: EnvironmentService) {}
+
+  async enrichCompany(domainName: string): Promise<CompanyInteface> {
+    const enrichedCompany = await axios.get(
```

Comment:
@FelixMalfait  Seems better to use https://docs.nestjs.com/techniques/http-module